corehq/apps/export/models/new.py

```diff
@@ -1548,7 +1548,12 @@ def get_value(self, domain, doc_id, doc, base_path, split_column=False, **kwargs
         )
         if not split_column:
             return value
+
         values = [None] * 4
+
+        if not isinstance(value, basestring):
+            return values
+
         for index, coordinate in enumerate(value.split(' ')):
             values[index] = coordinate
```

Comment:
note that this will still fail hard if for some reason there's more than 4 values returned from the split, but i figured that should be the case (let me know if not)